src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,22 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from "../../types/Todo";
+import classNames from "classnames";
+import { UserInfo } from "../UserInfo";
+
+type Props = {
+  todo: Todo
+}
+
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, user} = todo;
+
+  return (
+    <>
+      <article className={classNames('TodoInfo', {'TodoInfo--completed': completed})}>
```

Comment:
You can write like this
```suggestion
      <article className={classNames(
      'TodoInfo',
      { 'TodoInfo--completed': completed },
    )}
    >
```